plugins/uploadimage/plugin.js

```diff
@@ -19,6 +19,12 @@
 		return String( input );
 	}
 
+	function supportedTypes( types ) {
+		types = types || [ 'jpeg', 'png', 'gif', 'bmp' ];
```

Comment:
If there is default value, it could be directly defined as `CKEDITOR.config.uploadImage_supportedTypes`.